editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-execution-scope.tsx

```diff
@@ -97,6 +97,7 @@ export function createExecutionScope(
   }
 
   let executionScope: MapLike<any> = {
+    React: React,
```

Comment:
Does this mean it's effectively imported even if it's not?

Reply:
For the sake of the Canvas, yeah, because the Canvas is still being forced to use the old style transform. I'm going to add a follow up ticket for this actually, because really this PR is doing part of the work and then using a workaround for the rest.